### DISCUSSION TITLE
Github workflow "Cannot read property 'includes' of undefined"

### DISCUSSION THREAD

**Discussion by Fl4m3Ph03n1x** [community/community#25951] [Actions]
<h2>Background</h2>
<p>I have a small elixir project and I am trying to use GithubWorkflow to have a nice pipeline and automate some actions.</p>
<h2>Problem</h2>
<p>However I can’t even get it started as I get the following error:</p>
<pre><code class="lang-auto"> Setup elixir
1s
    AUCTION_HOUSE_TOKEN: ***
Run actions/setup-elixir@v1
  with:
    elixir-version: 1.10-otp-22
    otp-version: 22.1
    install-hex: true
    install-rebar: true
  env:
    COOKIE: ***
    TOKEN: ***
##[error]Cannot read property 'includes' of undefined
</code></pre>
<h2>Code</h2>
<p>I think my workflow might be incorrect, could someone help figure out what’s wrong?</p>
<pre><code class="lang-auto">name: build

on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]

env: 
  COOKIE: ${{ secrets.COOKIE }}
  TOKEN: ${{ secrets.TOKEN }}

jobs:

  build:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - name: Setup elixir
      uses: actions/setup-elixir@v1
      with:
        elixir-version: '1.10-otp-22' # Define the elixir version [required]
        otp-version: '22.1' # Define the OTP version [required]
    - name: Install Dependencies
      run: mix deps.get
    - name: Run Tests
      run: mix test
    - name: Run credo code analyser
      run: mix credo --strict
    - name: Run typechecks
      run: mix dialyzer
    - name: Build executable
      run: mix escript.build
      env: 
        MIX_ENV: prod
</code></pre>

**Comment by NicoNBW** [comment, marked as answer]
<p>Hi <a class="mention" href="/Fl4m3Ph03n1x">@Fl4m3Ph03n1x</a><br>
It seems that the <code>elixir-version</code> parameter is incorrect. Please try to specify the correct version, such as <code>1.10.0</code> version.</p>
<pre><code class="lang-auto"> build:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - name: Setup elixir
      uses: actions/setup-elixir@v1
      with:
        elixir-version: '1.10.0' # Define the elixir version [required]
        otp-version: '22.1' # Define the OTP version [required]
</code></pre>
<p>Please refer to this <a href="https://github.com/elixir-lang/elixir/releases" rel="nofollow noopener">doc</a> for more details about elixir version</p>
